docs/concepts.md

```diff
@@ -226,17 +226,20 @@ In ForestFlow this is referred to as the [BasicServeRequest](https://github.com/
      }
      ``` 
  
-   - <a name="serve-request-path">**path**</a>: Where is the model artifact stored (excluding the model file name). ForestFlow must be able to access this file path.
-     Example:
-     Local filesystem: file:///my-models/DreamWorks/schedule/0
-     Git: git@github.com:<USER or Org>/<project>.git#v<Numeric Contract Number>.<Release Version>
+   - <a name="serve-request-path">**path**</a>: Where the model artifact is stored (excluding the model file name). ForestFlow must be able to access this file path.
+
+     Examples:
+     - Local filesystem: `file:///my-models/DreamWorks/schedule/0`
+     - Git: `git@github.com:<USER or Org>/<project>.git#v<Numeric Contract Number>.<Release Version>`
+     - S3: `s3::https://s3-us-west-2.amazonaws.com:8082/my-bucket-1 bucket=my-bucket-1 region=us-west-1`
      
      ForestFlow currently supports the following [StorageProtocols](https://github.com/ForestFlow/ForestFlow/tree/master/core/src/main/scala/com/dreamworks/forestflow/utils/SourceStorageProtocols.scala):
```

Comment:
@gopikrishna967 can you please update this link in the docs to point to the correct git repo now that it moved off of DreamWorks' internal github to a public one?

Thanks!